packages/@sanity/desk-tool/src/panes/document/inspectDialog/InspectDialog.styles.tsx

```diff
@@ -0,0 +1,142 @@
+import {rem, Theme} from '@sanity/ui'
+import styled, {css} from 'styled-components'
+
+export const JSONInspectorWrapper = styled.div(({theme}: {theme: Theme}) => {
+  const {color, fonts, space} = theme.sanity
+
+  return css`
+    & .json-inspector,
+    & .json-inspector .json-inspector__selection {
+      font-family: ${fonts.code.family};
+      font-size: ${fonts.code.sizes[2].fontSize}px;
+      line-height: ${fonts.code.sizes[2].lineHeight}px;
+      color: var(--card-code-fg-color);
+    }
+
+    & .json-inspector .json-inspector__leaf {
+      padding-left: ${rem(space[4])};
```

Comment:
just me learning: is this the preferred way to add spacing from the design system?

Reply:
No, the preferred way is to use the props like `<Box padding={4}>` when possible.

However, in this case, the elements are rendered by a 3rd party library, so we need to set padding using CSS.

Ideally, `@sanity/ui` would have CSS variables for this (e.g. `--space-4`), but that is still work-in-progress.

So yeah, this is the way to use the spacing scale in custom CSS currrently :-)